src/test/java/uk/gov/register/functional/LoadSerializedFunctionalTest.java

```diff
@@ -26,17 +25,26 @@
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.junit.Assert.assertThat;
-import static uk.gov.register.functional.db.TestDBSupport.*;
 
 public class LoadSerializedFunctionalTest {
-    public static final int APPLICATION_PORT = 9000;
     private static final String registerName = "register";
 
     @Rule
     public TestRule wipe = new WipeDatabaseRule();
 
     @ClassRule
-    public static final RegisterRule register = new RegisterRule(registerName);
+    public static final RegisterRule register = new RegisterRule();
```

Comment:
Do you think it may be worth extracting a base class at this point or not yet because of a reason? Somink like this or is it too stinky and they accidentally look the same? Just a food for thought ;) no hate just love...
```
public abstract class BaseTest {
    @ClassRule
    public static final RegisterRule register = new RegisterRule();

    protected final CanonicalJsonMapper canonicalJsonMapper = new CanonicalJsonMapper();
    protected static TestRecordDAO testRecordDAO;
    protected static TestEntryDAO testEntryDAO;
    protected static TestItemCommandDAO testItemDAO;

    @BeforeClass
    public static void setUpClass() throws Exception {
        Handle handle = register.handleFor("register");
        testRecordDAO = handle.attach(TestRecordDAO.class);
        testEntryDAO = handle.attach(TestEntryDAO.class);
        testItemDAO = handle.attach(TestItemCommandDAO.class);
    }
}
```